packages/core/src/card/Card.tsx

```diff
@@ -23,7 +31,7 @@ export interface CardProps extends ComponentPropsWithoutRef<"div"> {
    */
   interactable?: boolean;
   /**
-   * Styling variant; defaults to "primary".
+   * Styling variant. Defaults to `primary`.
```

Comment:
This should be aligned to Input
```suggestion
   * Styling variant. Defaults to "primary".
```
If you want to use backticks you still need quotes to signify it's a string
```suggestion
   * Styling variant. Defaults to `"primary"`.
```